Documentation/op-guide/failures.md

```diff
@@ -30,7 +30,7 @@ Once a majority of members works, the etcd cluster elects a new leader automatic
 
 ## Network partition
 
-A network partition is similar to a minor followers failure or a leader failure. A network partition divides the etcd cluster into two parts; one with a member majority and the other with a member minority. The majority side becomes the available cluster and the minority side is unavailable; there is no “split-brain” in etcd. 
+A network partition is similar to a minor followers failure or a leader failure. A network partition divides the etcd cluster into two parts; one with a member majority and the other with a member minority. The majority side becomes the available cluster and the minority side is unavailable; there is no “split-brain” in etcd.  ( no “split-brain” is possible because etcd cluster members are explicitly added/removed, such that all cluster members know the target cluster size and can therefore determine majority. ) 
```

Comment:
to be very precise: "cluster members are explicitly added/removed" -> "cluster members are explicitly added/removed and each such change is approved by the current majority"

Reply:
Thanks for the feedback.  Tweaked.